src/components/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -90,48 +95,54 @@ export default defineComponent({
 @import '@/styles/variables';
 @import '@/styles/functions';
 
-.segmented-control .k-button {
-  --KButtonPrimaryBase: var(--blue-100);
-  --KButtonPrimaryHover: var(--blue-100);
-  color: var(--KSegementedControlPrimary, var(--blue-500, color(blue-500)));
-  border-radius: 0;
-  margin-left: -1px;
-  flex: 1;
-
-  &.primary {
-    z-index: 1;
-    border-color: var(--blue-500);
-  }
+.segmented-control {
+  column-gap: var(--KSegmentGap, 0px);
```

Comment:
I believe instead of `column-gap` this should just be `gap` in case the segments wrap to a new line (unsure if that's even possible)